### PR TITLE
(Docs) Update Readme and Changelog for 1.18.1

### PR DESCRIPTION
**Pull request by IgorCATech**
Signed-off-by: Igor Kazmyr <igor.kazmyr@broadcom.com>

## Proposed changes

The regular doc update before the next release. This is a patch release.

## Release Notes

Milestone:

Changelog: 1.18.1

## Types of changes

What types of changes does your code introduce to Zowe Explorer?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Updates to Documentation or Tests (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This checklist will be used as reference for both the contributor and the reviewer_

- [x] I have read the [CONTRIBUTOR GUIDANCE](https://github.com/zowe/vscode-extension-for-zowe/wiki/Best-Practices:-Contributor-Guidance) wiki
- [x] PR title follows [Conventional Commits Guidelines](https://www.conventionalcommits.org/en/v1.0.0-beta.2/)
- [x] PR Description is included
- [ ] gif or screenshot is included if visual changes are made
- [ ] `yarn workspace vscode-extension-for-zowe vscode:prepublish` has been executed
- [ ] All checks have passed (DCO, Jenkins and Code Coverage)
- [ ] I have added unit test and it is passing
- [ ] I have added integration test and it is passing
- [ ] There is coverage for the code that I have added
- [ ] I have tested it manually and there are no regressions found
- [x] I have added necessary documentation (if appropriate)
- [ ] Any PR dependencies have been merged and published (if appropriate)

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: packages/zowe-explorer/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,14 @@
 
 All notable changes to the "vscode-extension-for-zowe" extension will be documented in this file.
 
+## `1.18.1`
+
+- Improved integration tests for greater efficiency [#1430](https://github.com/zowe/vscode-extension-for-zowe/pull/1430). Thanks @katelynienaber
+- Introduced improvements for @dependabot. Bumped path-parse from 1.0.6 to 1.0.7 [#1441](https://github.com/zowe/vscode-extension-for-zowe/pull/1441). Thanks @lauren-li
+- Added a reminder to the PR template to create a PR from `master` to `next` after PR is merged into `master` branch [#1446](https://github.com/zowe/vscode-extension-for-zowe/pull/1446). Thanks @JillieBeanSim
+- Added cli dependency to api [#1445](https://github.com/zowe/vscode-extension-for-zowe/pull/1445). Thanks @phaumer
```
Comment (lauren-li) [inline packages/zowe-explorer/CHANGELOG.md:10]: Capitalize "CLI" and "API".

File: packages/zowe-explorer/README.md [hunk 2]
```diff
@@ -23,31 +23,29 @@ More information:
...
+## What's new in Zowe Explorer 1.18.1
 
-**Added**:
+Added:
 
-- Added the ability to register custom profile types in `ProfilesCache` for extenders.
-- Added the ability to pass account and other information from tso profile.
-- Add profiles cache to extenders.
+- Added a reminder to the PR template to create a PR from `master` to `next` after PR is merged into `master` branch.
+- Added cli dependency to API.
```
Comment (lauren-li) [inline packages/zowe-explorer/README.md:39]: Capitalize "CLI".

**Review by lauren-li — CHANGES_REQUESTED** [review]
I requested some minor capitalization changes, but aside from that, I think this PR looks good. However, I was wondering if we want to include the updates that aren't really user facing (e.g. the Dependabot update). I think we can leave that off, but I don't feel very strongly about it if others want to include it.

Thanks @IgorCATech for putting this together!

**Review by lauren-li — APPROVED** [review]
Looks good to me! Thanks @IgorCATech for all the updates!

**Review by jellypuno — APPROVED** [review]

**Review by crawr — APPROVED** [review]

**Pull request merged**